libs/common/parameters_utils.h

```diff
@@ -164,6 +166,10 @@ static inline bool VtValueGetBool(const VtValue& value, bool defaultValue = fals
         VtArray<int> array = value.UncheckedGet<VtArray<int>>();
         return array.empty() ? false : (array[0] != 0);
     }
+    if (value.IsHolding<VtArray<unsigned int>>()) {
+        VtArray<unsigned int> array = value.UncheckedGet<VtArray<unsigned int>>();
+        return array.empty() ? false : (array[0] != 0);
```

Comment:
should it return the defaultValue instead of false ? the other VtValueGet functions might need to be updated to return the defaultValue